source/ota_over_mqtt_demo.c

```diff
@@ -80,6 +76,10 @@
     #error "Please define the democonfigCLIENT_IDENTIFIER with the thing name for which OTA is performed"
 #endif
 
+#if !defined( MQTT_AGENT_NETWORK_BUFFER_SIZE ) || ( MQTT_AGENT_NETWORK_BUFFER_SIZE < OTA_DATA_BLOCK_SIZE )
```

Comment:
As this is OTA demo why not just set the buffer size from the ota data block size instead of throwing error?

Reply:
MQTT_AGENT_NETWORK_BUFFER_SIZE is used by all demos sharing the MQTT agent connection. Buffer size is set irrespective of OTA demo task.